rubocop/README.md

```diff
@@ -0,0 +1,46 @@
+# Carbon Five Rubocop Conventions
+
+We like and recommend Rubocop, with the caveat that a handful of its rules can be overly strict out of the box. We've found that development teams benefit from relaxing some of the default settings in order to remove unnecessary friction. This directory contains the configuration we consider standard practice for all of our internal Ruby code bases and a good starting point for starting greenfield client projects.
+
+## Installation
+
+### Making your own copy (recommended!)
+
+The safest way to use the standard Carbon Five Rubocop configuration is to copy all the `.rubocop*.yml` files from this repo into your project. Here is a quick way to do it with `wget`:
+
+```
+cd my-rails-app
+wget https://raw.githubusercontent.com/carbonfive/c5-conventions/main/rubocop/.rubocop{,-common,-performance,-rails,-rspec}.yml
```

Comment:
I tested this out (substituting the name of this PR branch for `main`) and it works great.